vendor/github.com/hashicorp/go-plugin/client.go

```diff
@@ -973,7 +973,22 @@ func (c *Client) logStderr(r io.Reader) {
 		entry, err := parseJSON(line)
 		// If output is not JSON format, print directly to Debug
 		if err != nil {
-			l.Debug(string(line))
+			// Attempt to infer the desired log level from the commonly used
```

Comment:
I think the comment two lines up from here is no longer correct, right? (Would've been better feedback on the go-plugin PR rather than this one, but :man_shrugging: ...)

Reply:
oops, thanks. That's what I get for only looking at the colored lines :/